docs/modules/extensions/pages/index.adoc

```diff
@@ -1,51 +1,89 @@
 = Extensions
 :url-exten-lab: https://github.com/asciidoctor/asciidoctor-extensions-lab
 
-Extensions are central to the success of AsciiDoc because they open up the language to new use cases.
-Asciidoctor provides an extension API that offers a superset of extension points.
-As a result, extensions in Asciidoctor are easy to write, powerful, and simple to distribute.
+////
+The code examples for this module are all in the examples directory.
+They are set up so that they may be run individually through the *-runner.sh script or tested all at once with the extensions-test.sh script.
+////
 
-Asciidoctor also allows extensions to be written using the full power of a programming language (whether it be Ruby, Java, Groovy or JavaScript).
-You don't have to shave yaks to get the functionality you want, and you can distribute the extension using defacto-standard packaging mechanisms like RubyGems or JARs.
+Extensions are central to the power of AsciiDoc because they open up the language to new use cases.
+Asciidoctor provides an extension API that offers a set of extension points.
```

Comment:
To me, that's exactly what my wording says.  I'll see if I can find a way to make it even clearer.

Reply:
How about...
```
Extensions are central to the power of AsciiDoc because they open up the language to new use cases.
To implement this critical functionality, Asciidoctor provides an extension API that offers a set of extension points.
```